java/ingest/ingest-batcher-core/src/main/java/sleeper/ingest/batcher/IngestBatcher.java

```diff
@@ -129,11 +132,11 @@ private boolean shouldCreateBatches(
                     inputFiles.size(), minFiles);
         }
         if (totalBytes < minBytes) {
-            LOGGER.info("Total bytes for files ({}) does not satisfy the minimum size for a job ({} bytes)",
+            LOGGER.info("Total size for files ({}B) does not satisfy the minimum size for a job ({}B)",
                     totalBytes, minBytes);
             meetsMinFiles = false;
         } else {
-            LOGGER.info("Total bytes for files ({}) satisfies the minimum size for a job ({} bytes)",
+            LOGGER.info("Total size for files ({}B) satisfies the minimum size for a job ({}B)",
                     totalBytes, minBytes);
```

Comment:
This is still using the raw bytes rather than including a human readable version.